nightly is broken

```
 WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
Building dependency tree...

Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 winehq-stable : Depends: wine-stable (= 8.0.1~jammy-1)
E: Unable to correct problems, you have held broken packages.
Error: Process completed with exit code 100.
```

--- a/.github/workflows/kind-nightly.yaml
+++ b/.github/workflows/kind-nightly.yaml
@@ -111,11 +111,14 @@ jobs:
 
       - name: Install wine
         run: |
-          sudo dpkg --add-architecture i386
-          wget -qO - https://dl.winehq.org/wine-builds/winehq.key | sudo apt-key add -
-          sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ jammy main'
           sudo apt-get update
-          sudo apt install --install-recommends winehq-stable
+          sudo apt-get install wget -y
+          sudo dpkg --add-architecture i386
+          sudo mkdir -pm755 /etc/apt/keyrings
+          sudo wget -O /etc/apt/keyrings/winehq-archive.key https://dl.winehq.org/wine-builds/winehq.key
+          sudo wget -NP /etc/apt/sources.list.d/ https://dl.winehq.org/wine-builds/ubuntu/dists/jammy/winehq-jammy.sources
+          sudo apt-get update
+          sudo apt-get install -y --install-recommends winehq-stable
 
       - name: Compile tkn-pac for windows
         run: |

